### PR TITLE
Jc #94 about us page

### PR DESCRIPTION
**Pull request by jdconrad89**
@case-eee @Robbie-Smith @kheppenstall @Dpalazzari @drod1000 styled both the about us page and the home page. I also pulled out the majority of the project index page into a partial so that we could show a few projects on the home page.

### CODE REVIEW

**Comment by jdconrad89** [comment]
screen shot 2017-02-15 at 4 31 04 pm
screen shot 2017-02-15 at 4 31 16 pm

basic views of the tops of the pages... the home page cycles through the pictures like a carousel

**Comment by Dpalazzari** [comment]
56 files!? LOL 😨

File: app/assets/stylesheets/sections/about.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        width: 100%;
+        border: 1px solid black;
+        word-wrap: break-word;
+        background: image_url("people_are_awesome.jpg") no-repeat center;
+        color: white;
+        text-shadow:
+        -1px -1px 0 #000,
+        1px -1px 0 #000,
+        -1px 1px 0 #000,
+    1px 1px 0 #000;
```
Comment (Dpalazzari) [inline app/assets/stylesheets/sections/about.scss]: @jdconrad89 Let's fix this indentation right here.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -21,8 +21,13 @@ def create
   def edit
     @user = found_user
   end
+
+  def password
+    byebug
```
Comment (Dpalazzari) [inline app/controllers/users_controller.rb]: @jdconrad89 Remove this por favor.

File: app/views/password/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div id='new_session'class="card col-md-offset-4">
+  <div class="card-block">
+    <h4 class='card-title text-center'>Enter the code that we just sent to your phone.</h4><br>
+    <div class="form-group text-center">
+      <%= form_for :password, url: update_password_path do |f|%>
+      <%= f.label :twilio_response_code %><br>
+      <%= f.text_field :twilio_response_code %><br>
+      <%= f.label :new_password %><br>
```
Comment (Dpalazzari) [inline app/views/password/edit.html.erb]: @jdconrad89 This only works for your phone number, right Jason?

**Comment by Dpalazzari** [comment]
@jdconrad89 I feel like I'm seeing double. Isn't a lot of this stuff in your other branch??

**Comment by jdconrad89** [comment]
@Dpalazzari a lot of the files are images that I added... there is a bit of carryover from when i merged this branch... why they're showing up on this PR i don't know. Don't you just love github.

**Comment by Dpalazzari** [comment]
@jdconrad89 I saw a couple minor things that can be adjusted that I commented on. I'm going to keep looking through this PR. We've merged this one with master and rspec'd right?

File: app/views/password/edit.html.erb (hunk 3 above)
Comment (jdconrad89) [inline app/views/password/edit.html.erb]: @Dpalazzari at the moment yes, if anyone else wants to use their number to test it i can add it to the verified numbers, I told Casey the same thing incase she wanted to use her phone during the eval tomorrow

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (jdconrad89) [inline app/controllers/users_controller.rb]: @Dpalazzari how does that keep popping up i've deleted that method and byebug like 4 times... Blast you github!

**Comment by jdconrad89** [comment]
@Dpalazzari just ran rspec again to double check... no issues... though we still have those 4 pending tests... have we fixed those?

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (jdconrad89) [inline app/controllers/users_controller.rb]: @Dpalazzari deleted

File: app/assets/stylesheets/sections/about.scss (hunk 1 above)
Comment (jdconrad89) [inline app/assets/stylesheets/sections/about.scss]: @Dpalazzari fixed... good catch don't know how i missed that indentation and things lining up is one of my few OCD's

**Comment by Dpalazzari** [comment]
@jdconrad89 We are going to leave those there. They are a relic and evidence that at one point we were successfully consuming our own API. It was just taking too long and timing out so we stopped and skipped the tests.

**Comment by jdconrad89** [comment]
@Dpalazzari makes sense

**Comment by Robbie-Smith** [comment]
@jdconrad89 Merge conflicts.

**Comment by jdconrad89** [comment]
@Robbie-Smith conflicts fixed

**Pull request merged**